documentation/rails.md

```diff
@@ -0,0 +1,124 @@
+Rails configuration
+===================
+
+To use this file with Rails you have two choices:
+
+#### 1. Adding a new `webpack_asset_url` helper
+
+You'll need a helper to replace asset pipeline. Here is an example how you an do this:
+
+```Ruby
+module WebpackHelper
+  def webpack_asset_url(asset)
+    "/assets/#{manifest.fetch(asset)}"
+  end
+
+  def manifest
+    @manifest ||= JSON.parse(File.read('manifest.json'))
+  rescue
+    fail 'Please run webpack'
+  end
+end
+```
+
+This file would then be saved in `app/helpers/webpack_helper.rb` for example.
```

Comment:
Reword. Maybe:

"This module can then be saved in an appropriately named file (e.g. `app/helpers/webpack_helper.rb`)."